### ISSUE TITLE
Small piece of code causes complete freeze

### ISSUE THREAD

**Issue by where-is-s** [microsoft/vscode#76697]
Issue Type: <b>Bug</b>

I encountered a piece of code which makes VS Code freeze indefinitely even without any extensions. Steps to reproduce:

1. Save this small piece code into a cpp file (for example crashtest.cpp):

```
DECLCALLBACK(void) test() {
    /* Test comment ******************************/
}
```

2. Open the file in VS Code

Expected result: VS Code allows me to edit the file.
Actual result: file opens but VS Code instantly freezes for indefinite time.

VS Code version: Code 1.36.0 (0f3794b38477eea13fb47fbe15a42798e6129338, 2019-07-03T13:20:56.241Z)
OS version: Linux x64 5.0.0-20-generic snap

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i5-8400 CPU @ 2.80GHz (6 x 3799)|
|GPU Status|2d_canvas: unavailable_software<br>flash_3d: disabled_software<br>flash_stage3d: disabled_software<br>flash_stage3d_baseline: disabled_software<br>gpu_compositing: disabled_software<br>multiple_raster_threads: enabled_on<br>native_gpu_memory_buffers: disabled_software<br>oop_rasterization: disabled_off<br>protected_video_decode: disabled_off<br>rasterization: disabled_software<br>skia_deferred_display_list: disabled_off<br>skia_renderer: disabled_off<br>surface_synchronization: enabled_on<br>video_decode: disabled_software<br>viz_display_compositor: disabled_off<br>webgl: unavailable_software<br>webgl2: unavailable_software|
|Load (avg)|1, 0, 0|
|Memory (System)|15.34GB (0.89GB free)|
|Process Argv||
|Screen Reader|no|
|VM|0%|
</details>Extensions: none

**Comment by matter123** [comment]
@where-is-s This should be fixed upstream with version 1.12.15 of marketplace.visualstudio.com/items?itemName=jeff-hykin.better-cpp-syntax.

if you want the changes immediately, install the Better C++ Syntax extension.
if you're okay with waiting a few days, you can use the VS Code Insider's release.
if you're okay with waiting a while, the changes will eventually be pulled into VS Code stable.

**Comment by alexr00** [comment]
@matter123 thank you for jumping on all these issues.
@where-is-s the issue will be fixed in 1.36.1!

**Comment by where-is-s** [comment]
> @where-is-s the issue will be fixed in 1.36.1!

Thank you @alexr00! I'm looking forward to that :)
